ecms_base/features/custom/ecms_emergency_notification/config/install/field.storage.node.field_emergency_weight.yml

```diff
@@ -0,0 +1,77 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - options
+id: node.field_emergency_weight
+field_name: field_emergency_weight
+entity_type: node
+type: list_string
```

Comment:
Since these are numbers, we should use `list_number` instead of string.

Reply:
It is changed to integers